docs/monitors/telegraf-statsd.md

```diff
@@ -46,11 +46,11 @@ Monitor Type: `telegraf/statsd`
 | `deleteCounters` | no | `bool` | Whether to clear the counter cache every interval.  Setting this to false means the cache will only be cleared when the monitor is restarted. (**default:** `true`) |
 | `deleteSets` | no | `bool` | Whether to clear the sets cache every interval.  Setting this to false means the cache will only be cleared when the monitor is restarted. (**default:** `true`) |
 | `deleteTimings` | no | `bool` | Whether to clear the timings cache every interval.  Setting this to false means the cache will only be cleared when the monitor is restarted. (**default:** `true`) |
-| `percentiles` | no | `list of integer` | The percentiles that are collected for timing and histogram stats. |
+| `percentiles` | no | `list of integers (see below)` | The percentiles that are collected for timing and histogram stats. |
```

Comment:
What is there to see below?

Reply:
Nothing, no idea why it's putting that in there.  The template is only supposed to do that if the element type is "object"